### ISSUE TITLE
[BUG]: 

### ISSUE THREAD

**Issue by Alistair1231** [martinrotter/rssguard#1133]
### Brief description of the issue

I created a python script to modify feeds and then output the result to stdout. It works great and wihtout issues. This is how I manually add the feed into RSS Guard.
![image](https://github.com/martinrotter/rssguard/assets/26165288/5ab513f9-990b-4f58-b0c5-a9ba729b58e9)

When I export an OPML however, I cannot import it afterwards, since it seems to treat my script as an URL and then of course fails. I think that, since I get the same error "protocol error", when I add a feed and forget to select Script as source before pressing "Fetch it now"

### How to reproduce the bug?

1. Add feed that uses a script
2. Export feed
3. Try to import the feed

### What was the expected result?

That I can import my feeds and use the OPML as a backup

### What actually happened?

It failed to apply the source as script and failed to import.

### Debug log

time="  2094.943" type="debug" -> network: Destroying SilentNetworkAccessManager instance.
time="  2094.944" type="critical" -> core: Cannot fetch medatada for feed: '"C:\ProgramData\miniconda3\python.exe" "C:\RSS Guard\scripts\script.py" "http://[url]" "another argument"' with error: 'protocol error'.

### Operating system and version

* OS: Win 11
* RSS Guard version: 4.5.1

### Example OPML that was exported

```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<opml version="2.0" xmlns:rssguard="https://github.com/martinrotter/rssguard">
  <head>
    <title>RSS Guard</title>
    <dateCreated>Sat, 14 Oct 2023 18:08:37 GMT</dateCreated>
  </head>
  <body>
    <outline description="" text="Ebooks">
      <outline description="[description]" encoding="UTF-8" rssguard:postProcess="" rssguard:xmlUrlType="1" text="[title text]" title="[title text]" type="rss" version="RSS" xmlUrl="[command]"/>
    </outline>
  </body>
</opml>

```

**Comment by martinrotter** [comment]
Yes, I see the error on source code. Will fix on monday, maybe tuesday.

**Comment by martinrotter** [comment]
Please post a screenshot of how your "import opml" dialog looks like when you import your opml, thanks.

@Alistair1231

**Comment by martinrotter** [comment]
likely fixed, wait for 4.5.2 release
